kodilla-advanced-tests/src/main/java/com/kodilla/mockito/homework/WeatherService.java

```diff
@@ -0,0 +1,50 @@
+package com.kodilla.mockito.homework;
+
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+public class WeatherService {
+    private Map<User,Set<Location>> users = new HashMap<>();
+
+    public void addUser(User user, Location location){
+        Set<Location> locations = users.getOrDefault(user,new HashSet<>());
```

Comment:
Po co ten argument pusty zbior w tej metodzie?

Reply:
if (users.get(user) != null) {
    users.get(user).add(location);
} else {
    users.put(user, Set.of(location));
}